src/pages/CompletedSurvey/ui/CompletedSurveyPage.module.scss

```
@@ -0,0 +1,50 @@
+@import 'src/app/styles/variables/global.scss';
+
+.CompletedSurveyPage {
+    padding-top: 21px;
+    display: flex;
+    gap: 22px;
+    flex-direction: column;
+    align-items: center;
+}
+
+.feedbackBlock {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 28px;
+}
+
+.imgContainer {
+    height: 232px;
+    width: 524px;
+}
```

если посмотреть на макет внимательно, то ты увидишь, что тут у image container'a есть отступ в 12px
```suggestion
  padding: 12px 0;
}
```